components/logged_in.jsx

```diff
@@ -31,10 +31,15 @@ export default class LoggedIn extends React.Component {
 
         // Device tracking setup
         var iOS = (/(iPad|iPhone|iPod)/g).test(navigator.userAgent);
+        var android = (/(Android)/g).test(navigator.userAgent);
```

Comment:
This could be done using [`UserAgent.isAndroid()`](https://github.com/mattermost/mattermost-webapp/blob/master/utils/user_agent.jsx#L70)

Reply:
@asaadmahmood agree with HH, you may do it like this as well:

add import,
```
import * as UserAgent from 'utils/user_agent.jsx';
```
Then have this code block,
```
        // Device tracking setup
        if (UserAgent.isIos()) {
            $('body').addClass('ios');
        } else if (UserAgent.isAndroid()) {
            $('body').addClass('android');
        }
```